classic-editor.php

```diff
@@ -622,14 +624,14 @@ public static function add_edit_links( $actions, $post ) {
 
 		// Link to the Block Editor.
 		$url = remove_query_arg( 'classic-editor', $edit_url );
-		$text = __( 'Block Editor', 'classic-editor' );
+		$text = __( 'Edit (Block Editor)', 'classic-editor' ); # TWEAK: more understandable description of the action of the link

```

Comment:
Was wondering about this too, but in Gutenberg the link was changed to "Classic Editor", so I copied it :)

Reply:
For me it is better: in en_CA / en_GB / it_IT and in other languages ​​I have translated it in this direction. Also Gutenberg, for me, has some obscure points, in some descriptions, see this for example: https://github.com/luciano-croce/gutenberg-translation-patch/

Reply:
Gutenberg text is no better... IMHO